suite-native/config/src/supportedNetworks.ts

```diff
@@ -69,9 +77,23 @@ export const portfolioTrackerMainnets = sortNetworks(
         .filter(network => !portfolioTrackerBlacklist.includes(network.symbol)),
 );
 
-export const portfolioTrackerTestnets = sortNetworks(
-    getTestnets().filter(network => networkSymbolsWhitelistMap.testnet.includes(network.symbol)),
-);
+const getPortfolioTrackerTestnets = () => {
+    const testnets = sortNetworks(
+        getTestnets().filter(network =>
+            networkSymbolsWhitelistMap.testnet.includes(network.symbol),
+        ),
+    );
+
+    // enable BTC REGTEST for debug and detox e2e builds
+    if (isDebugEnv() || isDetoxTestBuild()) {
```

Comment:
Is there some reason why not to allow regtest in all builds same as testnets?

Reply:
I did it on purpose because I could not imagine that someone would ever use regtest on any other build (develop, production) which should be running on a physical device. Since you need to have `trezor-user-env` running simultaneously, this setup appears quite cumbersome to me.

Do you think someone would ever use it? If yes, I can change it. But then it would make sense to hide the regtest behind some feature flag to not confuse users of the production app that never heard of a "regtest" network.

Reply:
I was thinking of using the areTestnetsEnabled flag, so the normal user wouldn't see it, but we would have it available for possible testing. The likelihood of using this is relatively small, but at the same time it seemed to me that it could be even simpler in code.

Reply:
Ok, i will change it then :)

Reply:
Feature flag added in [a6ce4ed](https://github.com/trezor/trezor-suite/pull/12116/commits/a6ce4ed44f7f094c70acd618dbc8ac224ef662e2)